core/interfaces/i_toolbox.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview The interface for a toolbox.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+'use strict';
+
+goog.provide('Blockly.IToolbox');
+
+goog.require('Blockly.IDisposable');
+goog.require('Blockly.IRegistrable');
+
+
+/**
+ * @extends {Blockly.IDisposable}
```

Comment:
Where is `Blockly.IDisposable` defined?

Reply:
Left in on accident. Should be removed from this PR.

Reply:
No worries, and glad you're thinking about moving that into its own interface (in a separate PR).